.github/workflows/hdfconverter-tests.yml

```diff
@@ -12,17 +12,15 @@ jobs:
 
     services:
       splunk:
-        image: splunk/splunk
+        image: splunk/splunk:9.1.2
```

Comment:
Do we have an issue or something on this repo to update this back to latest when the upstream issue is fixed?

Reply:
Nothing other than me keeping the issue tab open to see if they eventually respond to / fix it

Reply:
https://github.com/mitre/heimdall2/issues/5499

Reply:
Thanks